src/main/java/hudson/plugins/logparser/LogParserPublisher.java

```diff
@@ -169,6 +169,9 @@ public void perform(Run<?, ?> build, FilePath workspace, Launcher launcher, Task
         public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
         private volatile ParserRuleFile[] parsingRulesGlobal = new ParserRuleFile[0];
         private boolean useLegacyFormatting = false;
+        private String colorWarning = "orange";
```

Comment:
You're right, I was looking at the filename only and thinking the colors were in the publisher and not the descriptor.